core/src/main/java/org/elasticsearch/search/sort/GeoDistanceSortBuilder.java

```diff
@@ -113,11 +181,18 @@ public GeoDistanceSortBuilder order(SortOrder order) {
         return this;
     }
 
+    /** Returns the order of sorting. */
+    public SortOrder order() { 
+        return this.order;
+    }
+
     /**
      * Not relevant.
+     *
+     * TODO should this throw an exception rather than silently ignore a parameter that is not used?
      */
     @Override
-    public SortBuilder missing(Object missing) {
+    public GeoDistanceSortBuilder missing(Object missing) {
```

Comment:
Looks like `FieldSortBuilder` is the only impl of SortBuilder that actually supports `missing`, so I'd try to delete this from the interface and move it there.